generic/uname.c

```diff
@@ -85,7 +85,7 @@ char* Generic_uname(void) {
 
       if (uname_result == 0) {
          size_t written = xSnprintf(savedString, sizeof(savedString), "%s %s [%s]", uname_info.sysname, uname_info.release, uname_info.machine);
-         if (!String_contains_i(savedString, distro) && sizeof(savedString) > written)
+         if (!String_eq(savedString, distro) && sizeof(savedString) > written)
```

Comment:
This is not case-insensitive IIRC.

Reply:
Yes. And the previous call allowed for substrings, which your new implementation also forbids.

Reply:
And FreeBSD seems to need that, so dropped that commit